rundeckapp/grails-app/i18n/messages.properties

```
@@ -1683,3 +1683,5 @@ page.users.updated.label=Updated
 page.users.lastjob.label=Last Job execution
 page.users.tokens.label=N\u00BA Tokens
 page.users.tokens.help=You can administrate the tokens in the User Profile page.
+
+development.mode.warning=[Development Mode] H2 recommended only for development and testing
```

```suggestion
development.mode.warning=[Development Mode] Usage of H2 database is recommended only for development and testing
```
Suggest a change to this text, in case "H2" is not obvious what it is referring to